adapters/rtbhouse/rtbhouse.go

```diff
@@ -52,6 +101,24 @@ func (adapter *RTBHouseAdapter) MakeRequests(
 	return requestsToBidder, errs
 }
 
+func getImpressionExt(imp openrtb2.Imp) (*openrtb_ext.ExtImpRTBHouse, error) {
```

Comment:
The `openrtb_ext.ExtImpRTBHouse` struct has more fields than `BidFloor`, do we plan to use the others inside your implementation of `MakeRequests` in the future? If not, as a small optimization, we could probably we could probably favor `jsonparser.GetInt()` over this approach:
```
func getImpressionExt(imp openrtb2.Imp) (int, error) {
    return jsonparser.GetInt(imp.Ext, "bidfloor")
}
```

Reply:
Yes, we plan to use other fields of `ExtImpRTBHouse ` in the future.